File: serve.rb
require 'webrick'
port = (ENV['PORT'] || 8899).to_i
server = WEBrick::HTTPServer.new(
  Port: port,
  DocumentRoot: '/Users/imoto/Documents/GitHub/roteiro-facil',
  Logger: WEBrick::Log.new($stderr, WEBrick::Log::ERROR),
  AccessLog: []
)
trap('INT') { server.stop }
server.start

File: serve.py
import os, sys
os.chdir('/Users/imoto/Documents/GitHub/roteiro-facil')
port = int(os.environ.get('PORT', 8899))
import http.server, socketserver
handler = http.server.SimpleHTTPRequestHandler
with socketserver.TCPServer(("", port), handler) as httpd:
    httpd.serve_forever()
